README.md

```diff
@@ -4,6 +4,40 @@
 [![Slack Status](https://img.shields.io/badge/Slack_Channel-neuroimaging-E01563.svg)](https://swcarpentry.slack.com/messages/CCJBHKCHZ)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/carpentries-incubator/SDC-BIDS-fMRI/gh-pages)
 
+## Background
+
+This is one sub-module within the [Neuroimaging cirriculum][neuro_cirriculum]. Visit the link to view all the modules associated with the Neuroimaging Carpentries program.
+
+**fMRI Analysis in Python** is a programme developed to faciliate reproducible analysis in the area of functional neuroimaging analysis. Python is emerging as a standard language of data analysis, visualization, and workflow building. More recently, it has rapidly been adopted by the neuroimaging community as a means of developing powerful open-source tools in favour of historially used opaque packages such as AFNI, FSL and SPM. In addition, the barrier to entry to python is low - meaning that you as the user can easily develop your own packages and contribute to the open-source codebase of neuroimaging!
+
+
+***
+
+The **fMRI Analysis in Python** is a workshop series started up via a collaboration between researchers and staff at the Centre for Addiction and Mental Health (Toronto, ON), the University of Western Ontario (London, Ontario), and McGill University (Montreal, Quebec).
+
+***
+
+## About the lesson
+
+This lesson covers fMRI imaging analysis from the basic steps of preprocessing and data cleaning, to running an analysis to explore connectivity patterns in the brain.
+
+### Episodes
+
+| Time | Episode | Question(s) Answered |
+| ---  | ---     | ---                  |
+||Setup|Download files required for the lesson|
+| 00:00 | 1. Exploring Preprocessed fMRI Data from fMRIPREP   | How does fMRIPrep store preprocessed neuroimaging data. How do I access preprocessed neuroimaging data                                     |
+| 00:25 | 2. Introduction to Image Manipulation using Nilearn | How can be perform arithmetic operations on MR images                                                                                                  |
```

Comment:
From "How can **be** perform arithmetic operations on MR images" to "How can **we** perform arithmetic operations on MR images"

Reply:
thanks for the catch!